### PR TITLE
remove the last of the avatar billboard code

### PR DESCRIPTION
**Pull request by birarda**
- remove old avatar billboard code

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6cfa71cb55dfae473535cbc5947d89e98e5e00e4 available at: https://gist.github.com/db986b97273905bc8a577e2714469a6f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7799.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7799.dmg

**Comment by cozza13** [comment]
@birarda . test plan?

**Comment by kevinmthomas-carpool** [comment]
testing

**Comment by birarda** [comment]
To test:

- Connect two avatars to the avatar mixer and confirm that everything works as expected. The billboard code was not being referenced so its removal should not have changed anything.

**Comment by kevinmthomas-carpool** [comment]
## QA Test
**PR Build:** 7799
**Testing Architecture:** 64-bit PC Desktop & 64-bit MacBook Pro
**Testing OS:** Windows 10 & OS X El Capitan
**Testing Locations:** music
**Additional Test Results/Observations:** Successful test.  Connected two avatars to the avatar mixer on the ```music``` domain.  The first on a PC and the second on a Mac both with the identical PR. Expected behavior as identity packets seem normal.  Attached screen shots from both PC and MAC and logs from both OS if necessary.
**Successful Logout/Crash:** Successful logout on both.
music 1 pc
music 2 pc
music 1 mac
music 2 mac
[hifi-log PC.txt](https://github.com/highfidelity/hifi/files/245818/hifi-log.PC.txt)
[hifi-log MAC.txt](https://github.com/highfidelity/hifi/files/245819/hifi-log.MAC.txt)

File: assignment-client/src/Agent.cpp [hunk 1]
```diff
@@ -290,7 +290,6 @@ void Agent::executeScript() {
     packetReceiver.registerListener(PacketType::BulkAvatarData, avatarHashMap.data(), "processAvatarDataPacket");
     packetReceiver.registerListener(PacketType::KillAvatar, avatarHashMap.data(), "processKillAvatar");
     packetReceiver.registerListener(PacketType::AvatarIdentity, avatarHashMap.data(), "processAvatarIdentityPacket");
-    packetReceiver.registerListener(PacketType::AvatarBillboard, avatarHashMap.data(), "processAvatarBillboardPacket");
```
Comment (Atlante45) [inline assignment-client/src/Agent.cpp]: We should probably rename PacketType::AvatarBillboard to unused in `PachetHeaders.h`

**Comment by ZappoMan** [comment]
Did we test old vs new client/server with this PR? What happens when an old client connects to a new mixer? it looks like the old Agents will broadcast billboard packets to new mixers... how do the mixers handle this? I assume that they probably print some log message about an unknown packet type or a packet with no registered receiver? Does that produce log spam? Or minimal messages?

**Comment by ZappoMan** [comment]
The one place the old code appears to have been sending billboard packets is in the case of an old AC/agent... We should test cross version compatibility between old AC/agents and the new mixer....

1) run the PR build as a sandbox
2) get a script that creates an avatar (like a robot/NPC script)
3) run that script in your domain scripts, but specify a pool so you can control which AC gets assigned
4) run a "production" AC from the command line, using "-t 2 -pool xxxx" where xxxx is the name of the pool you used.
5) you should see the AC fire up, connect and start running the script...
6) verify the avatar-mixer AC start acting funny... acting funny would likely be: spewing log messages about unknown packets or something....

**Comment by kevinmthomas-carpool** [comment]
## QA Re-Test
**PR Build:** 7799
**Testing Architecture:** 64-bit PC Desktop
**Testing OS:** Windows 10
**Testing Locations:** localhost
**Additional Test Results/Observations:** Failure.  Loaded Sandbox and opened the setting page of the domain.  Loaded ```https://raw.githubusercontent.com/highfidelity/hifi/5042561e80367ee2ebefa7d56728246e2d037e9d/script-archive/dancing_bot.js``` and assigned it a pool named ```albert```.  Verified that it loaded in the node section as a pool awaiting an AC call.  Opened a terminal and navigated to c:/Program Files/High Fidelity and ran ```assignment-client.exe -t 2 -pool albert``` to which a crash ensued.  Log file included for deeper analysis.  Screen shots attached.  
**Successful Logout/Crash:** Crash.
albert pool
albert pool
crash
[hifi-log.txt](https://github.com/highfidelity/hifi/files/250839/hifi-log.txt)

**Comment by kevinmthomas-carpool** [comment]
## QA Re-Test
**PR Build:** 7799
**Testing Architecture:** 64-bit PC Desktop
**Testing OS:** Windows 10
**Testing Locations:** localhost
**Additional Test Results/Observations:** Partial Success.  Loaded Sandbox and opened the setting page of the domain.  Loaded ```https://raw.githubusercontent.com/highfidelity/hifi/5042561e80367ee2ebefa7d56728246e2d037e9d/script-archive/dancing_bot.js``` and assigned it a pool named ```albert```.  Verified that it loaded in the node section as a pool awaiting an AC call.  Opened a terminal and navigated to c:/Program Files/High Fidelity PR 7799 and ran ```assignment-client.exe -t 2 -pool albert``` to which a crash ensued.  AC on main release utilizing the folder "High Fidelity" worked successfully.  Verified ```albert``` pool was running on the nodes tab of the domain-server settings page.  Unsuccessful in seeing the dancing bot appear however.  Upon shutting down the AC, a command window crash occurred however Interface did not crash.  Log file attached for additional analysis.
Log file included for deeper analysis.  Screen shots attached. 
**Successful Logout/Crash:** Successful logout.
[hifi-log.txt](https://github.com/highfidelity/hifi/files/257948/hifi-log.txt)

**Comment by birarda** [comment]
That's a very old script. Can you test with this script:

https://gist.githubusercontent.com/birarda/1c60dc857b9019341e18727f337296f8/raw/f353d2ba5c84bcbac43b1877af2da037cd09563f/simplest-avatar.js

I've done tests with this locally just now, from an AC from this PR and from master and I do not get any crashes.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2cf943b2b16e9ed618d8e3704b8bde763388b35f available at: https://gist.github.com/db986b97273905bc8a577e2714469a6f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7799.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7799.dmg

**Comment by kevinmthomas-carpool** [comment]
## QA Re-Test
**PR Build:** 7799
**Testing Architecture:** 64-bit PC Desktop
**Testing OS:** Windows 10
**Testing Locations:** localhost
**QA PASS**
**Additional Test Results/Observations:** Loaded Sandbox and opened the setting page of the domain.  Loaded ```https://gist.githubusercontent.com/birarda/1c60dc857b9019341e18727f337296f8/raw/f353d2ba5c84bcbac43b1877af2da037cd09563f/simplest-avatar.js``` and assigned it a pool named ```simple```.  Verified that it loaded in the node section as a pool awaiting an AC call.  Opened a terminal and navigated to c:/Program Files/High Fidelity PR 7799 and ran ```assignment-client.exe -t 2 -pool simple``` to which a successful pool occurred.  Screen shot enclosed.  Render rate was slow however that was tied to an earlier bug which is now fixed and had no bearing on this PR.
**Successful Logout/Crash:** Successful logout.
pr7799

**Pull request merged**